cypress/TestCases/FireboltCertification/Accessibility.feature

```diff
@@ -1,157 +1,155 @@
-         | Set speed-2           | set speed as 2        | 2 for speed in voiceGuidance settings   | onvoiceGuidanceSettings with 2 for speed   |
+    @Accessibility @coreSDK @sdk @transport
+    Scenario Outline: Accessibility.audioDescriptionSettings - Positive Scenario: <Scenario>
+        Given we test the 'AUDIODESCRIPTIONS_SETTINGS' getters and setters '<Method>' to '<Value>'
+        When '3rd party app' registers for the 'Firebolt' event
+        When '3rd party app' invokes the 'Firebolt' get API
+        Given 1st party app invokes the 'Firebolt' API to set '<Method>' to '<Value>'
+        And 'Firebolt' platform responds to '1st party app' set API
+        When '3rd party app' invokes the 'Firebolt' get API
+        And 'Firebolt' platform responds to '3rd party app' get API
+        And 'Firebolt' platform triggers '3rd party app' event
 
-   @Accessibility @coreSDK @sdk @transport
-   Scenario Outline: Accessibility.audioDescriptionSettings - Positive Scenario: <Scenario>
-      When '3rd party app' registers for the 'accessibility onAudioDescriptionSettingsChanged' event using the 'Firebolt' API
-      And '3rd party app' invokes the 'Firebolt' API to 'get audioDescriptionSettings settings'
-      And 1st party app invokes the 'Firebolt' API to '<Key>'
-      Then 'Firebolt' platform responds to '1st party app' for '<Key>'
-      When '3rd party app' invokes the 'Firebolt' API to 'get audioDescriptionSettings settings'
-      Then 'Firebolt' platform responds with '<Method_Content>'
-      And 'Firebolt' platform triggers event '<Event_Content>'
+        Examples:
+            | Scenario                         | Method  | Value |
+            | Disable audioDescriptionSettings | enabled | false |
+            | Enable audioDescriptionSettings  | enabled | true  |
 
-      Examples:
-         | Scenario                         | Key                      | Method_Content                     | Event_Content                            |
-         | Disable audioDescriptionSettings | disable audioDescription | disabled audioDescription settings | onaudioDescriptionSettings with disabled |
-         | Enable audioDescriptionSettings  | enable audioDescription  | enabled audioDescription settings  | onaudioDescriptionSettings with enabled  |
+    @Accessibility @coreSDK @sdk @transport
+    Scenario Outline: Accessibility.closedCaptions - Positive Scenario: <Scenario>
+        Given we test the 'CLOSEDCAPTIONS' getters and setters '<Method>' to '<Value>'
+        When '3rd party app' registers for the 'Firebolt' event
+        Given 1st party app invokes the 'Firebolt' API to set '<Method>' to '<Value>'
+        And 'Firebolt' platform responds to '1st party app' set API
+        When '3rd party app' invokes the 'Firebolt' get API
+        And 'Firebolt' platform responds to '3rd party app' get API
+        And 'Firebolt' platform triggers '3rd party app' event
 
-   @Accessibility @coreSDK @sdk @transport @requiresPlatformImplementation
```

Comment:
Do we not need a clear listeners scenario anymore?

Reply:
Currently clear listeners glue for v2 not there, will take care in new ticket